addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -39,7 +41,7 @@ def send_mail_test(self):
             # Returns a proper error if there is a syntax error with Qweb
             body = mailing._render_field('body_html', record.ids, post_process=True)[record.id]
             preview = mailing._render_field('preview', record.ids, post_process=True)[record.id]
-            full_body = mailing._prepend_preview(body, preview)
+            full_body = mailing._prepend_preview(body if isinstance(body, Markup) else Markup(body), preview)
```

Comment:
You can simplify to
```suggestion
            full_body = mailing._prepend_preview(Markup(body), preview)
```
but having a `Markup(Markup())` somewhere is kind of weird

Reply:
Done.
Thanks.

Reply:
@mart-e The security check is still failing. I suppose it will be ignored.

Reply:
@kitan191 the check is not failing, it's warning you are using dangerous (but not forbidden) code, asking for an override as double validation. Sometimes you have to use it, just need a second :eyes: to make sure it's safe (which should be the case here)